index.bs

```diff
@@ -346,20 +346,99 @@ interface mixin DatagramTransport {
    but large enough to avoid dropping packets when for the stream is not read
    for short periods of time (due to the reader being paused).
 
-# `WebTransport` Mixin #  {#web-transport}
+# `WebTransport` Interface #  {#web-transport}
 
-The `WebTransport` includes the methods common to all transports, such as
-state, state changes, and the ability to close the transport.
+The `WebTransport` provides a unified interface to all client-server transports
```

Comment:
`The WebTransport` reads strangely to me. Maybe write `The WebTransport constructor` or just `WebTransport`?

Reply:
Done.